src/app/components/playback/playback_control.rs

```diff
@@ -110,13 +110,14 @@ pub struct PlaybackControl {
     clock: Clock,
 }
 
+const STEP: f64 = 5000.0;
 impl PlaybackControl {
     pub fn new(model: PlaybackControlModel, widget: PlaybackControlWidget) -> Self {
         let model = Rc::new(model);
         let debouncer = Debouncer::new();
         let debouncer_clone = debouncer.clone();
-
         let track_position = &widget.track_position;
+        widget.seek_bar.set_increments(STEP, STEP);
```

Comment:
I don't know the default values but I feel like the page size should be greater than the regular step size!

Reply:
The defaults are both 0. From documentation:
 "The step size is used when the user clicks the GtkScrollbar arrows or moves GtkScale via arrow keys. The page size is used for example when moving via Page Up or Page Down keys"
I think in this case it works out fine to have it at the same value. Maybe in the future have it as a setting that users can change?

Reply:
fair enough! :)